pkg/snapstore/snapshot.go

```diff
@@ -65,7 +65,7 @@ func ParseSnapshot(snapPath string) (*brtypes.Snapshot, error) {
 	}
 
 	lastIndex = lastIndex + 3
-	prefix := snapPath[:lastIndex]
+	prefix := snapPath[:lastIndex-1]
```

Comment:
It was decided to not touch legacy code in this PR, since it might have unintended consequences.
Addressed in [ff3b90d](https://github.com/gardener/etcd-backup-restore/pull/710/commits/ff3b90dc69c5d8d3d02c782d17f58da3cdd3776e).